examples/llava_example.py

```diff
@@ -23,13 +24,11 @@ def run_llava_pixel_values():
         "\nUSER: What is the content of this image?\nASSISTANT:")
 
     # This should be provided by another online or offline component.
-    image = torch.load("images/stop_sign_pixel_values.pt")
+    image = Image.open("images/stop_sign.jpg")
 
     outputs = llm.generate({
-        "prompt":
-        prompt,
-        "multi_modal_data":
-        MultiModalData(type=MultiModalData.Type.IMAGE, data=image),
+        "prompt": prompt,
+        "multi_modal_data": ImagePixelData(image),
     })
```

Comment:
Should we add an example here that if `--disable-image-processor` is specified, then we need to actually pass in pixel values (i.e, `images/stop_sign_pixel_values.pt`)?

Reply:
Sure!